queries/nix.scm

```diff
@@ -0,0 +1,146 @@
+;; Statements
+
+[
+    (binding)
+] @statement
+;; This matches assert statements as stand-alone statements. This
+;; is because it's common to have a series of assert statements in a file, which
+;; would otherwise all match as a single statement. See the playground/nix/asserts.nix
+;; file for an example.
+(assert_expression
+    "assert" @statement.start
+    condition: (_)
+    ";" @statement.end
+    body: (_)
+)
```

Comment:
are there tests for asserts?

Reply:
Not yet, I wanted to ask other peoples opinion about defining it as a standalone statement before I added a test, as the test implies it works the way we expect, but not sure what other people will expect.

Reply:
cc/ @auscompgeek

Reply:
See previous thread about removing `body: (_)` for more details on what I'm debating about